tests/Gedmo/Tree/MaterializedPathODMMongoDBRepositoryTest.php

```diff
@@ -103,22 +99,20 @@ public function testGetChildren(): void
         static::assertSame('Vegitables', $result->current()->getTitle());
 
         // Get direct children from the root, including it
-        /** @var CachingIterator $result */
         $result = $this->repo->getChildren($root, true, 'title', 'asc', true);
 
+        static::assertInstanceOf(CachingIterator::class, $result);
         static::assertSame(3, \iterator_count($result));
         $result->rewind();
-        $result->rewind();
         static::assertSame('Food', $result->current()->getTitle());
         $result->next();
         static::assertSame('Fruits', $result->current()->getTitle());
         $result->next();
         static::assertSame('Vegitables', $result->current()->getTitle());
 
         // Get direct children from the root, NOT including it
-        /** @var CachingIterator $result */
         $result = $this->repo->getChildren($root, true, 'title', 'asc', false);
-        static::assertInstanceOf(Iterator::class, $result);
+        static::assertInstanceOf(CachingIterator::class, $result);
```

Comment:
I think we should stick to `Doctrine\ODM\MongoDB\Iterator\Iterator` in these `static::assertInstanceOf()` calls since [`CachingIterator` is internal](https://github.com/doctrine/mongodb-odm/blob/69e64e7eb51f07bcfc09bc8e7ea8329f301b44ff/lib/Doctrine/ODM/MongoDB/Iterator/CachingIterator.php#L26)

Reply:
Done!